scalgrafanalib/core.py

```diff
@@ -0,0 +1,71 @@
+from typing import Any, Dict, TypeVar
+import attr
+from grafanalib import core  # type: ignore
+
+Json = Dict[str, Any]
+Self = TypeVar("Self")
+
+
+@attr.s
+class Tooltip(core.Tooltip):
+    """SimpleTooltip : simple, "modern" tooltip configuration
+    Inherit from Tooltip to allow using in place of "usual" tooltip class"""
+
+    show: bool = attr.ib(default=True, validator=attr.validators.instance_of(bool))
+    showHistogram: bool = attr.ib(  # pylint: disable=invalid-name
+        default=True, validator=attr.validators.instance_of(bool)
+    )
+
+    def to_json_data(self) -> Json:
+        return {"show": self.show, "showHistogram": self.showHistogram}
+
+
+@attr.s
```

Comment:
Do we need the decorator when the class has no attributes, just the method redefinition?